romtools/workflows/greedy/run_greedy.py

```diff
@@ -294,7 +294,8 @@ def reg(val):
                 training_samples=training_samples,
                 fom_time=fom_time,
                 rom_time=rom_time,
-                basis_time=basis_time)
+                basis_time=basis_time,
+                parameter_samples=parameter_samples)
```

Comment:
I don't think this is the right place , Eric was pointing to line 202. This is writing the sample inside the stats at the end , while we want to write them "on the fly".

Reply:
Resolved